crates/simplexpr/src/ast.rs

```diff
@@ -86,6 +86,39 @@ impl SimplExpr {
     pub fn synth_literal<T: Into<DynVal>>(s: T) -> Self {
         Self::Literal(s.into())
     }
+
+    pub fn dfs_collect_var_refs_into(&self, dest: &mut Vec<VarName>) {
```

Comment:
what does the `dfs` stand for here 🤔 just `collect_var_refs_into` should be enough

Reply:
`dfs` stands for `Depth-first search` just a name to define what type of traversal going to happen while collecting the refs, though is not necessary to know cuz its just implementation detail, I'll just rename it 👍.